html/config.html

```diff
@@ -0,0 +1,666 @@
+                        myObj[key][key2][key3] = "yes";
+                      } else {
+                        myObj[key][key2][key3] = "no";
+                      };
+                    } else if (key3 == "feeds"){ // *****Unused in this version*****
+                      feeds = myObj["plugins"]["rss"]["feeds"];
+                      Object.keys(feeds).forEach(function(key4) {
+                        myObj[key][key2][key3][key4]["name"] = document.getElementById("rss_feed" + key4 + "_name").value;
+                        myObj[key][key2][key3][key4]["url"] = document.getElementById("rss_feed" + key4).value;
+                        if (document.getElementById("rss_feed" + key4 + "_on").checked == true) {
+                          myObj[key][key2][key3][key4]["enabled"] = "yes";
+                        } else {
+                          myObj[key][key2][key3][key4]["enabled"] = "no";
+                        }
+                      });
+                    } else { // Save all other plugin variables
+                      myObj[key][key2][key3] = document.getElementById(key2 + "_" + key3).value;
+                    };
+                  };
+                });
+              });
+            };
+          });
+          // Save all new config values to config.json
+          if (task == "save") {
+            var xhr = new XMLHttpRequest();
+            var url = "upload";
+            xhr.open("POST", url, true);
+            xhr.setRequestHeader("Content-Type", "application/json");
+            var data = JSON.stringify(myObj);
+            xhr.send(data);
+            alert("Update successful.") // Figure out error checking before alerting "success"
+          };
+        });
+      }
+    </script>
+    <title>PiWeatherRock Configuration</title>
+    <link rel="stylesheet" href="style.css" type="text/css" />
+  </head>
+  <body><!--- onload="config_helper('load')">--->
+    <div class="myDiv">
+    <div class="bg"></div>
+      <div id="lcloud"><img src="chancetstorms.png" height="256" width="256"></div>
+      <div id="rcloud"><img src="chancetstorms.png" height="256" width="256"></div>
+      <div class="page_title">PiWeatherRock Configuration Page</div>
+      <div id="version"></div>
+      <p class="pad"></p>
+<!--***** unused in this version      <div id="start_toggle">
+        <div class="left">Stop</div>
+        <label class="switch">
+          <input type="checkbox" id="start" onclick="startstop()">
+          <span class="slider round"></span>
+        </label>
+        <div class="right">Start</div>
+      </div>
+-->      
+      <!-- Rounded switch -->
+      <div id="toggle">
+        <div class="left">Basic</div>
+        <label class="switch">
+          <input type="checkbox" id="advanced" onclick="hide('advanced', 'advanced_config')">
+          <span class="slider round"></span>
+        </label>
+        <div class="right">Advanced</div>
+      </div>
+      <p></p>
```

Comment:
Why the empty paragraphs? There seem to be a several of them.

Reply:
Left over code when I was attempting to fix spacing issues. Those do need to be removed. I've also found a way to save the correct primitive types in the config file (int, bool, string, etc). But now I need to amend the code to load it properly.

Reply:
Found CSS solution